samples/smp_svr/zephyr/src/main.c

```diff
@@ -1,3 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
```

Comment:
Is it OK to add a new license **over** an existing license?

Reply:
The licence is the same and I left original copyright. It is just that this was initially copied from zephyr which doesn't use explicit licence header but only SPDX identifier.

I'm open for suggestion.  Removing SPDX and leaving only copyright notice is also an option. Or maybe put all under same comment /*  */ block..

Reply:
The copyright is actually wrong anyway, Wind River did not contribute that `main.c`. So I would just get rid of everything below this line.

Reply:
OK, I'll update PR